packages/react-components/src/stack/stack.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react/no-array-index-key */
+import React, {
+  Children,
+  Fragment,
+  cloneElement,
+  forwardRef,
+  isValidElement,
+  useMemo
+} from 'react';
+import type * as Stitches from '@stitches/react';
+import { styled, config } from '../stitches.config';
+import { Box } from '../box';
+import { getSeparatorStyles } from './get-separator-style';
+
+const StackWrapper = styled(Box, {
+  display: 'flex'
+});
+
+export type MediaKeys = keyof typeof config.media;
+type Directions = 'row' | 'column' | 'row-reverse' | 'column-reverse';
```

Comment:
we can replace this with `React.CSSProperties['flexDirection']`

Reply:
@maneeshp97  this is causing properties such as inherit, initial of flex to be assignable which is not present in `stackDirections` which can be only row, column, row-reverse and column-reverse, hence causing the build to fail.

Reply:
Can we omit inherit and initial from `React.CSSProperties['flexDirection']` . Will that solve the problem ?